codegen/swagger/pom.xml

```diff
@@ -35,10 +35,13 @@
                   <packageName>bollard-stubs</packageName>
                   <packageVersion>1.49.3-rc.28.4.0</packageVersion>
```

Comment:
Perfect, let's increment this to `packageVersion` `1.49.4-rc.28.4.0` and rebuild, we can then merge this and make a release of bollard-stubs. Subsequently we can add endpoints against the generated models.